sklearn/utils/sparsefuncs.py

```diff
@@ -25,6 +26,49 @@ def _raise_error_wrong_axis(axis):
         raise ValueError(
             "Unknown axis value: %d. Use 0 for rows, or 1 for columns" % axis)
 
+def csr_csc_min_axis0(X):
```

Comment:
Unfortunately, the minimum scipy version that scikit-learn doesn't have the `min` functions for `csr` and `csc` sparse matrices.